README.md

```diff
@@ -18,42 +19,49 @@
 </p>
 
 The [end-2-end build](https://github.com/ondat/discoblocks/blob/main/.github/workflows/e2e-on-pr.yml) includes:
+
 - [gosec scanning](https://github.com/ondat/discoblocks/blob/main/.github/workflows/_gosecscan.yml)
 - [golang-ci linting](https://github.com/ondat/discoblocks/blob/main/.github/workflows/_gocilint.yml)
 - [Docker image build](https://github.com/ondat/discoblocks/blob/main/.github/workflows/_docker-build.yml)
 - [Trivy vulnerability scanning](https://github.com/ondat/discoblocks/blob/main/.github/workflows/_trivy.yml)
 
 -----
 
-**Please note**: We take security and users' trust seriously. If you believe you have found a security issue in Discoblocks, *please responsibly disclose* by following the [security policy](https://github.com/ondat/discoblocks/security/policy). 
+**Please note**: We take security and users' trust seriously. If you believe you have found a security issue in Discoblocks, *please responsibly disclose* by following the [security policy](https://github.com/ondat/discoblocks/security/policy).
 
 -----
 
 This is the home of [Discoblocks](https://discoblocks.io), an open-source declarative disk configuration system for Kubernetes helping to automate CRUD (Create, Read, Update, Delete) operations for cloud disk device resources attached to Kubernetes cluster nodes.
 
-* Website: https://discoblocks.io 
-* Announcement & Forum: [GitHub Discussions](https://github.com/ondat/discoblocks/discussions)
-* Documentation: [GitHub Wiki](https://github.com/ondat/discoblocks/wiki)
-* Recording of a demo: 
+- Website: [https://discoblocks.io](https://discoblocks.io)
+- Announcement & Forum: [GitHub Discussions](https://github.com/ondat/discoblocks/discussions)
+- Documentation: [GitHub Wiki](https://github.com/ondat/discoblocks/wiki)
+- Recording of a demo: [Demo](https://user-images.githubusercontent.com/55788733/168624989-c9b1d469-d3e5-40e7-8858-c9ff4a5b5428.mp4)
+
+## About the name
+
+Some call storage snorage because they believe it is boring... but what we could have fun and dance with the block devices!
```

Comment:
I think this line makes more sense, if you remove the "what". i.e
> Some call storage snorage because they believe it is boring... but we could have fun and dance with the block devices!